index.html

```diff
@@ -16,10 +16,19 @@
   <header>
   <nav>
     <h3 class="logo" id="logo"><a href="#"> Aliza Amir</a></h3>
+    <div class="menu">
+      <span id="hamburger-button">&#9776</span>
+      <span class="hidden" id="cross-button">X</span>
+    </div>
+    <ul class="menue-popup hidden">
+      <li class="menue-items"><a href="#portfolio-link" title="navbar link portfolio">Portfolio</a></li>
+      <li class="menue-items"><a href="#about-link" title="navbar link About">About</a></li>
+      <li class="menue-items"><a href="#Contact-link" title="navbar link Contact">Contact</a></li>
+    </ul>
```

Comment:
- [ ] Kindly add more space on the menu list from the top just as it is in the wireframe. At the moment, the menu looks cluttered and is aligned to the same line as the close X icon. Please follow the design guideline as much as you can. 👍🏽
- [ ] Please add more space between the menu items from the top and the icon. See the screenshots below: 👍🏽

**Your Design**

<img width="458" alt="Screenshot 2023-05-18 at 8 29 52 PM" src="https://github.com/alizaamir123/mobile-version/assets/50941074/796fbb58-1c6b-458d-87c3-95d79622b3ab">


**The Original Template**

<img width="302" alt="Screenshot 2023-05-18 at 8 32 10 PM" src="https://github.com/alizaamir123/mobile-version/assets/50941074/0dab2e72-ecea-46c8-a039-d5f3c22299db">